src/OSVRTrackedDevice.cpp

```diff
@@ -118,18 +118,19 @@ vr::EVRInitError OSVRTrackedDevice::Activate(uint32_t object_id)
     // file, use an empty dictionary instead. This allows the render manager
     // config to zero out its values.
     if (configString.empty()) {
-        logger_->Log("OSVRTrackedDevice::OSVRTrackedDevice(): Render Manager config is empty, using default values.\n");
+        logger_->Log("OSVRTrackedDevice::Activate(): Render Manager config is empty, using default values.\n");
         configString = "{}";
     }
 
     try {
         m_RenderManagerConfig.parse(configString);
     } catch(const std::exception& e) {
-        const std::string msg = "OSVRTrackedDevice::OSVRTrackedDevice(): Exception parsing Render Manager config: " + std::string(e.what()) + "\n";
+        const std::string msg = "OSVRTrackedDevice::Activate(): Exception parsing Render Manager config: " + std::string(e.what()) + "\n";
         logger_->Log(msg.c_str());
     }
 
-    return vr::VRInitError_None;
+	logger_->Log("OSVRTrackedDevice::Activate(): Activation complete.\n");
```

Comment:
clang-format, per the CONTRIBUTING file (there's one in this repo, right? if not, see another OSVR repo) should do the trick.